pennylane_lightning/lightning_gpu/_adjoint_jacobian.py

```diff
@@ -61,8 +67,23 @@ class LightningGPUAdjointJacobian(LightningBaseAdjointJacobian):
 
     # pylint: disable=too-few-public-methods
 
-    def __init__(self, qubit_state: LightningGPUStateVector, batch_obs: bool = False) -> None:
+    def __init__(
+        self,
+        qubit_state: LightningGPUStateVector,
+        batch_obs: bool = False,
+        use_mpi: bool = False,
+        mpi_handler=None,
+    ) -> None:
+
         super().__init__(qubit_state, batch_obs)
+
+        self._dp = DevPool()
```

Comment:
DO we need to initialize the DevPool even for `use_mpi = False`?

Reply:
yes, it's required for batch_obs support

Reply:
yes, [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/4bd44b182078d47f27739f9e8589dbaff0fe4f54/pennylane_lightning/lightning_gpu/_adjoint_jacobian.py#L195)